File: frameworks/Python/django/hello/hello/urls.py
from django.urls import path
from world.views import plaintext, json, db, dbs, fortunes, update

urlpatterns = [
    path("plaintext", plaintext),
    path("json", json),
    path("db", db),
    path("dbs", dbs),
    path("fortunes", fortunes),
    path("update", update),
]

File: frameworks/PHP/unic/application/app/urls.php
<?php
//Include views
require_once 'view.php';

//URLs routing
$urlpatterns = [
  '/plaintext' => 'view.plaintext',
  '/json' => 'view.json',
  '/db' => 'view.db',
  '/queries' => 'view.queries',
  '/updates' => 'view.updates',
  '/fortunes' => 'view.fortunes',
];
